OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -115,6 +119,13 @@ public bool InterpretCommand(S server, Connection conn, Session.Client client, s
 							return true;
 						}
 
+						if (server.Settings.DisableSinglePlayer &&
+							server.LobbyInfo.Clients.Where(c => c.Bot == null && c.Slot != null).Count() == 1)
+						{
+							server.SendOrderTo(conn, "Message", "Unable to start the game because singleplayer games are disabled on this server.");

```

Comment:
"Unable to start the game until another player joins" would be a bit more concise and matches the message above.